README.md

```diff
@@ -63,6 +63,10 @@ Flags:
       --links.validate.without-address-regex=^$  
                                  If specified, all links will be validated,
                                  except those matching the given target address.
+      --links.validate.without-github-links=""  
```

Comment:
Hm, this is great, but I wonder if we can introduce better composability here. 

E.g what if we find good mechanism for understanding GH wiki links, notifications or releases?

What if someone has website X and knows that links like a/b/c will work there as long as host is ok? 

I think we should allow some flexibility for users to add those, WDYT?

I would suggest something like:

```
--links.validate.config="<yaml content of filepath>"
```
(`pathorcontent` flag) with yaml like:

```
validate:
  validators:
     - regex: "(^http[s]?:\/\/)(www\.)github\.com"
       type: "github"
     # by default things are checked with `type: roundtrip`

```

.. so similar flow as with `globs` and transform yaml WDYT?

Reply:
I think that would be great! It would be better to allow users to pass in yaml config and maybe have multiple validators. Will try and implement this!